packages/gatsby-theme/src/blocks/ShareBlock/ShareBlock.stories.mdx

```diff
@@ -0,0 +1,93 @@
+import { Meta, Story, Props, Preview } from "@storybook/addon-docs/blocks"
+import { useArgs } from "@storybook/client-api"
+import _ from "lodash"
```

Comment:
Since we are using lodash directly, we should add it to our `package.json`. What do you think? @luisduenas

Reply:
I agree